cmds/script.c

```diff
@@ -133,6 +133,7 @@ int command_script(int argc, char *argv[], struct opts *opts)
 	struct uftrace_data handle;
 	struct uftrace_task_reader *task;
 	struct script_info info = {
+		.script_version = SCRIPT_VERSION,
```

Comment:
API version?  I don't want to repeat "script" in the script_info..

Reply:
I've changed it to `api_version`.